src/java/devcontainer-feature.json

```diff
@@ -33,10 +33,20 @@
       "default": false,
       "description": "Install Gradle, a build automation tool for multi-language software development"
     },
+    "gradleVersion": {
+      "type": "string",
```

Comment:
Could you please add a `proposals` array (like jdkDistro).  

I think it would be cool to also add another version other than latest as an example of what the options should look like.

Reply:
I agree that it would be great to keep `default: "latest"`